misc/vtr-libs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('vpr-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: vtr-libs
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/verilog-to-routing/vtr-verilog-to-routing.git
+  git_rev: master
+  patches:
+    - osx.patch             [osx]
+
+build:
```

Comment:
I wonder if we could leverage conda outputs, to have a single build for vtr that ends up in multiple conda packages.

Maybe @ajelinski could advise on the best way to consolidate this?